### PR TITLE
RF: ZNC-26: Issue Template to highlight Discourse

### PR DESCRIPTION
**Pull request by LaureVergeron**
Following conversations with Jono Bacon, it was highlighted that we should insist on redirecting people to the forum on the issue submission form. This pull request attempts to make a stronger statement than before on this matter.

### CODE REVIEW

**Comment by smaffulli** [comment]
This is good to go.

**Comment by LaureVergeron** [comment]
Thanks @stefano, thought I fixed those... squashed both commits and rebased, ready to merge @rahulreddy

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -49,6 +71,19 @@ What currently happens
...
-### Use case
+### Usecase
 
 Please provide use cases for changing the current behavior
+
+### Additional information
+
+- Is this request for your company? Y/N
+
+..- If Y: Company name:
```
Comment (rahulreddy) [inline .github/ISSUE_TEMPLATE.md]: Same comment as in other projects. If this is intended to be a sub-bullet please use two spaces before the `-`.

Comment (LaureVergeron) [reply .github/ISSUE_TEMPLATE.md]: Fixed, thanks

**Pull request merged**